playbooks/roles/backup-swift-container/meta/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 dependencies:
-  - tarsnap
+  - install_tarsnap
```

Comment:
From what I see here, `install_tarsnap` is just a tag, right, and `tarsnap` is the role? AFAIK [from the documentation](https://docs.ansible.com/ansible/2.5/user_guide/playbooks_reuse_roles.html#role-dependencies), you cannot use tags here, you need to use actual role names.

Reply:
I just checked the role name will be `install` as per [this](https://github.com/open-craft/ansible-playbooks/pull/62/files#diff-7adcc8954ef0db37fa83c1a33ee2543b) file. I could have kept it to `tasks/tarsnap.yml` , but I guess on the task, @itsjeyd suggested [so](https://tasks.opencraft.com/browse/OC-4845?focusedCommentId=78914&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-78914).

What we can do is keep to tasks namely `tarsnap` and `backup_tarsnap`. @UmanShahzad is this right?

Reply:
@UmanShahzad : I messed this up. Thanks for pointing it out. Let me update the whole PR.
Btw, how do I test this?

Reply:
@taranjeet That's alright :) Testing locally will probably require you spin up a VM with Vagrant or similar, and configure your Ansible environment to run playbooks on it, which is nontrivial. It may be easiest to test on some stage server we have, e.g. RabbitMQ, but that may require significant access. I will let your reviewer @itsjeyd figure this part out with you.